pkg/pub_worker/lib/src/bin/pana_wrapper.dart

```diff
@@ -167,7 +167,10 @@ Future<void> main(List<String> args) async {
     docDir: rawDartdocOutputFolder.path,
   );
 
-  await rawDartdocOutputFolder.delete(recursive: true);
+  // existence check is needed as post-processing may delete this directory
```

Comment:
When this method creates the folder, this method should probably also delete it.